### ISSUE TITLE
Survey Questions for CFA 

### ISSUE THREAD

**Issue by kcoronel** [hackforla/admin#68]
### Overview
Create a survey to share with Hack for LA members that helps us gauge how people feel about CFA's announcement of having a different relationship with the Brigade Network in 2023. The biggest change being that CFA will no longer be a  fiscal sponsor of Brigades. Other changes include:

- **Transition to Independent Brigades**: Code for America will be transitioning away from handling Brigade finances, as we believe there are better options at this point to help Brigades establish financial independence.
- **Refresh Formal Relationship Structure**: Together with newly independent Brigades, we will explore new ways of supporting you and being in relationship with each other.
- **Partner on Digital Equity**: In 2023, we will build upon the work and success of the [Digital Equity collective action event 8](https://discourse.codeforamerica.org/t/collective-action-broadband-action-track-details-and-updates/1491) at last year’s Brigade Congress to improve outreach and enrollment in the Affordable Connectivity Program. We hope this becomes a model for how we can partner in targeted ways moving forward.

### Action Items
- [x] Create survey 
- [x] Create first message for Hack for LA Slack that will be posted with survey link, survey will close at 7pm PST Wednesday, February 8, 2023
- [x] Create a reminder message to complete survey for Hack for LA Slack 
- [x] Create a survey closing message for Hack for LA Slack
- [ ] Compile questions received into one document and add to resources below
- [ ] Send questions to CFA 
- [x] Message HackforLA Slack on 2/13 for one last call for people to respond to survey

### Resources/Instructions
- ~[Link to Survey: Admin](https://docs.google.com/forms/d/1D_-67skaGZvCoJXMDzwqbNbN5Bty5bRp_2CnHUxiEuE/edit)~
- ~[Public Survey Link](https://docs.google.com/forms/d/e/1FAIpQLSfX5hmKMhHzuHxCh3f7pg6Hc4qFi6PTDylM7x63bi9IzY5eZQ/viewform)~
- [Link to Survey: Admin](https://docs.google.com/forms/d/1UVJkMHpFUr6ib-QMNbNKEuNnlV5advB7YFfnly-RLjk/edit)
- [Public Survey Link](https://docs.google.com/forms/d/e/1FAIpQLSe4Py19lEEWxAleAru2g38aQEphbTfSYv-7DS5j3kAKIp3A4g/viewform)
- Link to Responses 
- [CfA end of fiscal sponsorship questions from survey](https://docs.google.com/document/d/1mffP92C7zSO3Xi3ftU3JYRLwH2opjHvS6x1hXo37ofo/edit)
- [CfA end of fiscal sponsorship questions from HfLA to CfA](https://docs.google.com/document/d/1T9p40Lz3VVD89yM9LIHYM91hnyVH-GU4q8aDgQ9Vokw/edit#heading=h.fpc2kvumkw45)
- [CFA FAQ](https://cfa.notion.site/Frequently-Asked-Questions-FAQ-c4f08102c60544828e9ea4501bf2bc65)
- [HfLA google drive migration to CTS](https://docs.google.com/document/d/1VZwkkwFu5hcXtkyKs1ZuV_7CwhyJO4Fu1JZAoI5zT5c/edit)

**Comment by ExperimentsInHonesty** [comment]
# Top of survey
### Summary
Code for America (CfA) is going to stop being the fiscal sponsor of Brigades.  Hack for LA will be transitioning from CfA to the non profit, Civic Tech Structure (CTS) that Bonnie setup in late 2021, in preparation for this change.

CfA has some structured meetings and will compile questions submitted in advance.  This survey will help us to understand what questions you would like included in our submission and the level of your interest in the topic of fiscal sponsorship transition, so that we can keep you informed to the level you are interested.

#### How we think projects will be effected
- Projects that use tools such as Mapbox that are currently receiving licenses through CfA, will need to transition to new licenses that will be negotiated between CTS and Mapbox.
- Google Workspace (drive, email, docs, sheets, presentations, etc.) is used by all projects and we will have to work with CfA and Google on a transition plan. 

#### How we think Hack for LA the organzation will be effected
This is fantastic news for Hack for LA!
- Fundraising will be easier, no confusion with large funders who think that giving money to Code for America means that Hack for LA receives money from them (we don't) 
- More clarity and autonmony about what we report and when
- Stronger relationships with other networks.  
  - CTS intends to become a chapter of Code for All (directly)
  - CTS intends to join the Open Knowlege network
- CTS's mission in restricted to Los Angeles, projects that Hack for LA has that will benifit other communities will benifit from this larger non regional affiliation

### Code For America's CEO announcement
From: Amanda Renteria, posted 2023-02-02

Dear CfA Brigades and Volunteers,

As we begin the year, I’m writing to share some reflections, appreciations, and forthcoming changes to the Network program at Code for America.

I’d like to start by acknowledging that the significance of the Brigade Network cannot be overstated. There is no doubt that Code for America’s Brigade Network has had an enormous impact on how government works. As I’ve stated before, we have always believed that people working locally, whether as volunteers, local leaders, or elected officials, all guided by and centering those who are most impacted by the systems we aim to change, are a key part of the civic tech movement.

And the potential of this community, created through the Brigade Network, is limitless. The volunteers, Brigades, and Fellows who have dedicated their time and talents to this movement have helped define Code for America and the civic tech space–proving that technology can be used for good at the local level and creating a network of alums who are serving in government today.

Meanwhile, Code for America has put a stake in the ground about where we believe we need to focus our strategy. Across all of our programs, we are intentionally pursuing work that is advancing equity and alleviating poverty through the mindful use of technology. We believe that this community of civic technologists is better positioned than ever to move our movement closer to that vision in cities and counties all across the country. We have some ideas based on our learnings over the years about how to best do that.

As we start the year, we’re clear-eyed about the realities of working at the local level, grateful to the contributions and leadership of Network staff, and hopeful for what is possible with a more focused approach in 2023 to make an even greater impact on people’s lives.

With sincere appreciation,

Amanda

**TL;DR**
Given how our work has evolved, and the decrease in funding available to support a decentralized network, in 2023, we will embark on a set of changes to our relationship with the Brigade Network that we believe will enable us to more efficiently collaborate in ways that deliver significant impact in advancing equity and alleviating poverty and enable Brigades to more easily sustain your own local initiatives. In 2023, we will do three things:


- **Transition to Independent Brigades**: Code for America will be transitioning away from handling Brigade finances, as we believe there are better options at this point to help Brigades establish financial independence.
- **Refresh Formal Relationship Structure**: Together with newly independent Brigades, we will explore new ways of supporting you and being in relationship with each other.
- **Partner on Digital Equity**: In 2023, we will build upon the work and success of the [Digital Equity collective action event 8](https://discourse.codeforamerica.org/t/collective-action-broadband-action-track-details-and-updates/1491) at last year’s Brigade Congress to improve outreach and enrollment in the Affordable Connectivity Program. We hope this becomes a model for how we can partner in targeted ways moving forward.
- For more detail of our thinking, strategic shifts, and next steps please see below:


**How Code for America’s Work Has Evolved Over Time**
Code for America’s vision has endured since its founding: that government can work for the people, by the people, in a digital age. Over the years, we’ve remained laser-focused on this vision while experimenting with the most efficient and effective way to achieve it. There is no doubt that Code for America’s Brigade Network has had an enormous impact on bringing this vision to life.

Alongside the work of the Brigade Network, in 2015 we began experimenting with supporting in-house staff to work shoulder-to-shoulder with community organizations and governments to build digital tools and services, change policies, and improve programs directly. This work has enabled us to help millions of clients and unlock billions in benefits–a scale that we only ever dreamed was possible.

As our impact via direct service grew, our organizational structure, ways of working, and core capacities shifted to support that type of delivery–and it has become harder for us to do that while providing effective and efficient support and fiscal sponsorship for a large, decentralized base of volunteers, especially without significant multi-year resources available to support the necessary infrastructure for Network convening and engagement.

**How Volunteer Engagement Has Evolved Over Time**
We know that Brigade leaders have felt signs of volunteer fatigue and a less connected Network, largely due to the distance, uncertainty, and challenges created by COVID. In 2021 and 2022, driven by this learning and a desire to bring our work with Brigades into closer alignment with our in-house direct service work, we invested more resources in our Network program than ever before to develop and implement the ReVisioning strategy.

We learned that this post-pandemic environment has challenged volunteer engagement for more reasons than just COVID. Volunteerism is down across industries. There are growing privacy and security concerns with volunteers partnering with governments. Also, it has become even more difficult to fundraise larger, multi-year investments for a national volunteer network because of these larger shifts in volunteerism, and we experienced a decrease in philanthropic support for the Brigade Network. This seems to be a major challenge all volunteer-driven non-profit organizations have today.

At the same time, we have seen how Brigade volunteers can play a key role in driving direct impact, like increasing the uptake of vital government services, when it is done in deep collaboration and partnership with Code for America and local organizations through programs like the 2022 Impact Sprints.

Some highlights include Code for Tulsa’s functioning text messaging-based service that keeps people updated about their court cases, Open Maine’s work to enhance election transparency by digitizing paper campaign finance forms, and Code for Boston’s work to improve access to the Air Source Heat Pumps program in underserved communities. We also see it in the work of volunteers who are engaged in our Get Your Refund program and the emerging effort to close the participation gap in the Affordable Connectivity Program.

**How We Propose To Partner with the Network to Advance Equity and Alleviate Poverty in 2023**
Given how our work has evolved, in 2023, we will embark on a set of changes to our relationship with the Brigade Network that we believe will enable us to collaborate in ways that leverage the best of what Code for America can offer from an organizational perspective, with an emphasis on partnering with local volunteers and organizations in ways that deliver significant impact in advancing equity and alleviating poverty through the mindful use of technology in cities and counties all across the country.

In 2023, we will do three things:

1. **Transition to Independent Brigades**: Over the next six months, Brigades will transition to independent entities, and we will work with each Brigade to explore options for more effective fiscal management. Some Brigades may elect to partner with one of the many great organizations whose primary mission is fiscal sponsorship. Others may elect to establish an independent 501(c)(3). Whatever the path, we will work with Brigades to support a smooth and successful transition to independence. A number of Brigades already operate financially independently and can provide models for transition.
2. **Refresh Formal Relationship Structure**: Together with newly independent Brigades, we will also explore new ways of being in relationship with each other that better align with our mission and respective interests while creating the space for you to more easily continue your own local initiatives. For example, we want to explore partnership agreements, marketing agreements, licensing agreements, or other methods for formalizing our work together that tap into our respective strengths and shared goals.
3. **Partner on Digital Equity**: In 2023, we aim to prototype solutions in lock-step with outreach partners and newly independent Brigades in up to 3 states to improve outreach and enrollment in the Affordable Connectivity Program, building upon the work and success of the [Digital Equity collective action event 8](https://discourse.codeforamerica.org/t/collective-action-broadband-action-track-details-and-updates/1491) at last year’s Brigade Congress. This Digital Equity effort is one way that Code for America will bring together volunteers, newly independent Brigades, and organizations in partnership to deliver impact for people. We hope this becomes a model for how we can partner in targeted ways moving forward. Will you join us?

**Staff Changes**
As we evaluated our program and development strategies, we recognized that we would need to eliminate three positions on the Network team to set ourselves up for long-term sustainability. This is never easy, and we want to share a note of gratitude to the staff whose positions were eliminated in this process, and their invaluable contributions to building, nurturing, and convening the broader Network over the years:

- Em Burnett has been a longtime member of the Brigade Network and joined the Network team in February 2021 and played a key role in the design and development of ReVisioning. Em, thank you for your inspiring facilitation, clear-eyed approach to problem-solving, and steadfast leadership. Em’s last day at Code for America is February 28.
- Jill Bjers has been a part of the Brigade community since 2013, having co-founded and led the Open Charlotte Brigade, serving as a member and chair on the National Advisory Council from 2016-2019, and participating in the 2019 Community Fellowship program. Jillzey joined Code for America in November 2021 and was the lead event producer for the 2022 Brigade Congress. Jillzey, thank you for all that you’ve poured into the Network. Jillzey’s last day at Code for America was January 31.
- Sung Kim joined Code for America in August 2022 and jumped right into the mix, designing and delivering on a national day of action in St. Louis that helped us deepen our understanding of the barriers to the Affordable Connectivity Program and the design choices that are causing a low participation rate in the important program. Sung’s last day at Code for America was January 31.
 
Finally, we want to thank Ben Treviño, Senior Program Director, Network, for his invaluable contributions to Code for America throughout his two-year tenure. Ben has received an opportunity he cannot pass up in Hawaiʻi with the Hawaiʻi Leadership Forum, and his last day is February 28th. We want to thank Ben for his leadership, vision, and the intentionality and care that he brings to his work and for his colleagues, volunteers, and partners.

Mari Anthonette Miranda, Jess Manapul, and Mellina Stoney on the Network team, with dedicated support from the Finance team, will spend the next 6 - 8 months working with individual Brigades to work on the efforts around fiscal sponsorship. Along parallel paths, the Network team, along with members of the executive team and current Brigade leaders will explore how we will shape the next iteration of this important relationship between Code for America and Brigades and volunteers and work that emerges related to Digital Equity and ACP. James Armes will support any technical partnerships with newly independent Brigades and volunteers related to ACP. Evonne will lead and support the team in her role as VP of Programs.

**Immediate Next Steps**
I know that this community has experienced a tremendous amount of change in the past few years and that this news may raise questions and reactions that we want to make space for in the coming weeks. There are a few ways for us to connect and share questions and thoughts. We will share an FAQ and hold a series of forums this month:

 February 9, 2023 4:00 PM : Immediate reactions and questions
 February 16, 2023 4:00 PM : Facilitated, open ended discussion
 February 23, 2023 4:00 PM : Report back and synthesis

**Comment by kcoronel** [comment]
# Messages for Slack: 

1/ Hi everyone! You may have heard Code for America's (CfA) announcement about changes to their relationship with the Brigade Network (which includes Hack for LA). The most impacting change being that CfA is going to stop being the fiscal sponsor of Brigades.  In preparation for this change, Hack for LA will be transitioning from CfA to the non profit, Civic Tech Structure (CTS) that Bonnie setup in late 2021. 

We want to hear from you about how you feel about this change and give you a space to ask any questions you may have at this time. Please complete this survey by **7pm PST Wednesday, February 8, 2023.** 

2/ Friendly reminder to complete the [Name of Survey] by **7pm PST Wednesday, February 8, 2023.**

3/ Last chance to let us know how your thoughts on Code for America's recent announcement to changes to their relationship with Brigades. The[Name of Survey] will be closing TONIGHT at **7pm PST Wednesday, February 8, 2023.**

**Comment by ExperimentsInHonesty** [comment]
### Keep me informed
#### Do you want to be informed of the Fiscal Sponsorship Transition process
YES
No

Collect Email 
Collect Name

**Comment by ExperimentsInHonesty** [comment]
### Transition Qs
#### What questions do you have for Code for America about the transition process

**Comment by ExperimentsInHonesty** [comment]
### End of CfA Fiscal Sponsorship
####  What questions do you have for Code for America about the ending of the brigade network fiscal sponsorship?

**Comment by ExperimentsInHonesty** [comment]
### Software Licenses
#### Do you use software/apis on your project that has licenses (like Mapbox).  Software and licenses not effected are: Figma, Miro and Zoom.

**Comment by ExperimentsInHonesty** [comment]
### CfA Logo usage
#### Where do you use the CfA logo
- Use CfA log on your website
- slide decks 
- other 

Use checkboxes

**Comment by kcoronel** [comment]
@ExperimentsInHonesty survey has been added to this issue

**Comment by ExperimentsInHonesty** [comment]
@kcoronel I created a new form (with a Hack for LA header) and some logic for skipping questions, and some additional questions that Dean suggested.  Can you review it and see if it works for you?

Hack for LA & Code for America changes survey form: [Edit](https://docs.google.com/forms/d/1UVJkMHpFUr6ib-QMNbNKEuNnlV5advB7YFfnly-RLjk/edit) | [Viewform](https://docs.google.com/forms/d/e/1FAIpQLSe4Py19lEEWxAleAru2g38aQEphbTfSYv-7DS5j3kAKIp3A4g/viewform)

**Comment by kcoronel** [comment]
@ExperimentsInHonesty Looks good to me and I like the addition of the questions!

**Comment by ExperimentsInHonesty** [comment]
I @ channel posted in the slack just now.

Bonnie
  [8:45 AM](https://hackforla.slack.com/archives/C04502L0P/p1675788320795739)
:mega: Hi everyone! You may have heard Code for America's (CfA) announcement about changes to their relationship with the Brigade Network (which includes Hack for LA).

The most impacting change being that CfA is going to stop being the fiscal sponsor of Brigades.  In preparation for this change, Hack for LA will be transitioning from CfA to the non profit, Civic Tech Structure (CTS) setup by Bonnie late 2021 in preparation for this eventuality.

:heavy_dollar_sign:We have provided more details (e.g., what it means for ) and have a place to get your  :memo:feedback or questions you want answered here: [Hack for LA & Code for America changes survey](https://docs.google.com/forms/d/e/1FAIpQLSe4Py19lEEWxAleAru2g38aQEphbTfSYv-7DS5j3kAKIp3A4g/viewform).  Please complete survey by 7pm PST Wednesday, February 8, 2023.

**Comment by kcoronel** [comment]
Posted the above message on HackforLA's LinkedIn page

**Comment by kcoronel** [comment]
Notes from Admin Meeting on 2/11 
**Next CFA meeting 2/16** 
- Compiling the responses from survey with the intent to send these questions to CFA before 2/14

**Comment by ExperimentsInHonesty** [comment]
I reviewed the responses about software and wrote to 
### UI/UX CoP
> [@Aparna Gopal](https://hackforla.slack.com/team/U0245UJP868) How often (per month) do you use CfA zoom to host the speakers on the community of practice?

### Access the data
> [@Judson](https://hackforla.slack.com/team/U9NFW84QK) You indicated in the form that one of the Software & API usage items you use is Drive and OSS.  Can you explain what you are referring to when you say OSS?

**Comment by kcoronel** [comment]
@ExperimentsInHonesty I've added the questions that members asked in our survey

**Comment by ExperimentsInHonesty** [comment]
# Teams that have filled out the form
- [ ] 311
- [x] Access the Data
- [x] BallotNav
- [ ] Brigade Organizer's Playbook
- [x] Civic Tech Jobs
- [ ] Design Systems
- [ ] Expunge-Assist
- [ ] Food Oasis
- [ ] Guides
- [x] HackforLA.org
- [ ] HomeUniteUs
- [x] Tech Work Experience (internship)
- [x] LA TDM
- [ ] Lucky Parking
- [ ] Open Community Survey
- [x] People Depot
- [x] VRMS
- [ ] Youth Justice Nav

**Comment by ExperimentsInHonesty** [comment]
We wrote to all teams that have not filled it out
- 311
- Design Systems
- Expunge-Assist
- Food Oasis
- Guides
- HomeUniteUs
- Lucky Parking
- Youth Justice Nav

# Bonnie will fill it out for these teams

- [ ] Open Community Survey

# Bonnie will reach out to Thad to get him to fill it out on behalf of this team
- [ ] Brigade Organizer's Playbook

**Comment by kcoronel** [comment]
# Teams that have filled out the form (updated 3/4/23)

- [x] 311
- [x] Access the Data
- [x] BallotNav
- [ ] Brigade Organizer's Playbook
- [x] Civic Tech Jobs
- [x] Design Systems
- [x] Expunge-Assist
- [x] Food Oasis
- [ ] Guides
- [x] HackforLA.org
- [ ] HomeUniteUs
- [x] Tech Work Experience (internship)
- [x] LA TDM
- [x] Lucky Parking
- [ ] Open Community Survey
- [x] People Depot
- [x] VRMS
- [ ] Youth Justice Nav

**Comment by ExperimentsInHonesty** [comment]
will CfA continue to provide access to the CfA slack, or will it be restricted to staff or staff+ the brigades/volunteers working on "Partner on Digital Equity". 
- If there are no plans to change the slack, who will manage the culling of channels (that seemed to be a yearly or bi yearly event
- If it will be restricted.  When will the existing channels be archived or deleted?

Based on their responses we might need to go and look at the history of channels such as BOP and Project Index, so that we can capture relevant content

**Comment by kcoronel** [comment]
**Email sent to Jessica Manapul on 3/7/22:** 

Hi Jess, 

Bonnie and I were unfortunately unable to make the last two town hall meetings regarding the updates to the Brigade network. We've kept up to date with all of the resources you've sent post meetings (thank you!) However, we still have a few specific questions related to HackforLA operations that we would like to discuss with you or the appropriate Brigade Network team member. We've gathered our questions in [this document](https://docs.google.com/document/d/1T9p40Lz3VVD89yM9LIHYM91hnyVH-GU4q8aDgQ9Vokw/edit), so that we can make sure our meeting is productive :) 

What is the best way to get on your schedule? 

Thank you! 

Karen

**Comment by kcoronel** [comment]
Response from Jessica Manapul: 

Hi all!

These are great questions. Thanks for putting this document together. Let me connect with our marketing team regarding logo usage, the Youtube channel, and the Brigade website (great suggestion, by the way). Then, I'll reach back out to schedule a time for us to chat!

In the meantime:
Key Dates: The Key Dates link should've actually directed you to the [Key Dates section](https://cfa.notion.site/Brigade-Offboarding-Process-efe513bdc47844febb89dd8d404293dc#f4697624878d4e679abb17a065af9bf0) on the main page on the Notion. It's been fixed, thanks for catching that!
MOU: There will not be a new MOU after the current MOUs end on June 30th. However, Brigades who wish to continue their work may choose to transfer to a new fiscal sponsor (like [Open Collective](https://discourse.codeforamerica.org/t/open-collective-foundation-as-a-brigade-fiscal-host-ask-ocf-staff-anything/1652)) or establish their own 501(c)3.
Google: Can you show me where, on your end, it says that Code for America is the owner of Hack for LA's Google Workspace? We don't own your [hackforla.org](http://hackforla.org/) domain and I don't see your email accounts tied to CfA's Google Workspace on my end. I will verify with our IT team.
Mapbox: Unfortunately, it doesn't look like there is a way to transfer access tokens between accounts, but I'll do some more digging here. Hack for LA will likely need to get their own account when you folks switch to a new fiscal sponsor or before the MOU ends on June 30th, whichever comes first. There is a [grace period](https://cfa.notion.site/Brigade-Offboarding-Process-efe513bdc47844febb89dd8d404293dc#eabad478724544b285d9f0807cf914c5) for in-kinds though, which we can talk about in our meeting.

Jess Manapul | she/her

Brigade Community Specialist, Network

[Code for America](https://codeforamerica.org/?utm_campaign=cfa-general&utm_medium=email&utm_source=staff-email-signature) | [Join our email list!](https://info.codeforamerica.org/newsletter?utm_campaign=newsletter-signup&utm_medium=email&utm_source=staff-email-signature)

**Comment by kcoronel** [comment]
@ExperimentsInHonesty should we close this issue?
